src/main/scala/com/simple/jdub/Row.scala

```diff
@@ -306,5 +306,18 @@ class Row(rs: ResultSet) {
     colNames.zip(colValues).toMap
   }
 
-  private def extract[A](f: A): Option[A] = if (rs.wasNull()) None else Some(f)
+  private[this] def extract[A](f: A): Option[A] = if (rs.wasNull()) None else Some(f)
+
+  def array[T: reflect.ClassTag](index: Int): Array[T] = extractArray[T](rs.getArray(index + 1))
+
+  def array[T: reflect.ClassTag](name: String): Array[T] = extractArray[T](rs.getArray(name))
+
+  private[this] def extractArray[T: reflect.ClassTag](sqlArray: java.sql.Array): Array[T] = {
+    if (rs.wasNull) Array.empty[T] else {
```

Comment:
Please *always* use braces.
Also, style in this file is to include parens on `wasNull`. I'd prefer consistency with that.